src/points/PointsIntro.tsx

```diff
@@ -0,0 +1,88 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { PointsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import BackButton from 'src/components/BackButton'
+import Button, { BtnSizes } from 'src/components/Button'
+import CustomHeader from 'src/components/header/CustomHeader'
+import { pointsIllustration } from 'src/images/Images'
+import { replace } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { pointsIntroDismissed } from 'src/points/slice'
+import { useDispatch } from 'src/redux/hooks'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.PointsIntro>
+
+export default function PointsHome({ route, navigation }: Props) {
+  const { t } = useTranslation()
+
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    const unsubscribe = navigation.addListener('beforeRemove', () => {
+      ValoraAnalytics.track(PointsEvents.points_intro_back)
+    })
+
+    return unsubscribe
+  }, [navigation])
+
+  const onIntroDismiss = () => {
+    ValoraAnalytics.track(PointsEvents.points_intro_dismiss)
+    dispatch(pointsIntroDismissed())
+    replace(Screens.PointsHome)
+  }
+
+  return (
+    <SafeAreaView style={styles.outerContainer} edges={['top']}>
```

Comment:
why not have both top and bottom edges? then maybe you don't need the paddingBottom on the container (which might be too small for some phones)

Reply:
good catch! i found it not necessary to specify any [`edges`](https://github.com/th3rdwave/react-native-safe-area-context#edges) here at all. defaults works just fine.